rocky/crisis_room/management/commands/dashboards.py

```diff
@@ -0,0 +1,87 @@
+import json
+import logging
+from datetime import datetime, timezone
+from pathlib import Path
+from typing import Any
+from uuid import uuid4
+
+from crisis_room.models import Dashboard, DashboardData
+from django.conf import settings
+from django.core.management import BaseCommand
+from tools.models import Organization
+from tools.ooi_helpers import create_ooi
+
+from octopoes.connector.octopoes import OctopoesAPIConnector
+from octopoes.models.ooi.reports import ReportRecipe
+from rocky.bytes_client import get_bytes_client
+from rocky.scheduler import ReportTask, ScheduleRequest, scheduler_client
+
+FINDINGS_DASHBOARD_NAME = "Crisis Room Findings Dashboard"
+
+
+def get_or_create_default_dashboard(
+    organization: Organization, octopoes_client: OctopoesAPIConnector | None = None
+) -> bool:
+    valid_time = datetime.now(timezone.utc)
+    created = False
+    path = Path(__file__).parent / "recipe_seeder.json"
+
+    with path.open("r") as recipe_seeder:
+        recipe_default = json.load(recipe_seeder)
+
+    dashboard, _ = Dashboard.objects.get_or_create(name=FINDINGS_DASHBOARD_NAME, organization=organization)
+
+    dashboard_data, created = DashboardData.objects.get_or_create(dashboard=dashboard)
+    if created:
```

Comment:
I think this If should not be there, and instead the actions following this should be idempotent (eg, can be ran more than once wiithout causing issues). 
(rollback on failures on all of these dashboard creation actions could be another strategy, but that seems more work to implement across services.)